packit/upstream.py

```diff
@@ -404,9 +404,16 @@ def fix_spec(self, archive: str, version: str, commit: str):
             g_desc_raw = git_des_out.rsplit("-", 2)[1:]
             # release components are meant to be separated by ".", not "-"
             git_desc_suffix = "." + ".".join(g_desc_raw)
+        try:
+            all_pr_list = self.local_project.git_service.get_project().get_pr_list()
+            current_branch = self._local_project.git_repo.active_branch
+            pr_id = [p.id for p in all_pr_list if p.source_branch == current_branch]
```

Comment:
What I propose to do here is to get the name of the current branch here:
```python
current_branch = run_command(["git", "rev-parse", "--abbrev-ref", "HEAD"], output=True).strip()
```
Then replace `/` with `.`, since packit names PR branches like this: `pr/123`.